### PR TITLE
Switch to SCSS and add Solarized Dark

### PR DESCRIPTION
**Pull request by agentydragon**
Hi!
I like to switch between light/dark themes. I rewrote the code of this theme to SCSS, replaces references to constant Solarized Light colors to their "symbolic names", and made it so that you can also generate a dark variant of the same theme.
When I tried the dark theme, I noticed a few spots that previously Solarized Light didn't actually reskin that jumped out in the dark version, since they remained light when I tried it out.

So, basically, with this PR, this also supports Solarized Dark (in the same CSS).

### CODE REVIEW

**Comment by agentydragon** [comment]
Unfortunately I didn't yet have time to fix CodeMirror in Solarized Dark:

image

**Comment by agentydragon** [comment]
Got Solarized working in CKEditor. However I haven't yet figured out the right CSS to change the highlight bg color, see screenshot.

image

**Comment by WKSu** [comment]
Hey, nice work! Thanks for this. Unfortunately I couldn't figure out the background CSS as well but maybe you can look at something like the [Allure theme](https://github.com/JadeVane/Allure) if they have it.
